WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift

```diff
@@ -15,8 +15,8 @@ class LoadingStatusView: UIView {
 
```

Comment:
Seems to me like the `accessibilityHint` is not relevant here anymore. Also seems like we are passing in an already localized string for the title so maybe re-localizing it is not necessary with localizedStringWithFormat? What do you think?

Reply:
Good Catch 💯 
Fixed at dbe48ce31b609337a4b47c0dee88a9c093432b0d